content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,6 +72,30 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/etcd
+
+Example: `apiserver.latency.k8s.io/etcd: "4.730661757s"`
+
+This annotation indiactes the measure of latency incurred inside the storage layer,
+it accounts for the time it takes to send data to the etcd and get the complete response back.
+
+Please note that it does not include the time incurred in admission, or validation.
+
+## apiserver.latency.k8s.io/decode-response-object
+
+Example: `apiserver.latency.k8s.io/decode-response-object: "450.6649ns"`
+
+This annotation records the decoding time of response object.
+
+## apiserver.latency.k8s.io/apf-queue-wait
+
+Example: `apiserver.latency.k8s.io/apf-queue-wait: ""`
```

Comment:
Why is this set to the empty string as an example; can we come up with a better example?

Reply:
sure , I'll add some thing while updating other ones.